docs/vm/cpu-pinning.md

```diff
@@ -0,0 +1,112 @@
+```
+systemReserved: 408 millicores
+kubeReserved: 612 millicores
+```
+In this example, approximately 15 cores (14980 millicores) are available for workloads.
+
+When a VM (pod) in the Guaranteed QoS class requests 4 CPUs, 4 cores are dedicated to that VM. Pods in the other QoS classes share the remaining 11 cores in the shared pool.
+
+## Enable and Disable CPU Manager
+
+When you enable the CPU Manager, Harvester sets the CPU Manager policy to `static`. When you disable the feature, Harvester switches the CPU Manager policy back to `none`. 
+
+You must enable or disable the CPU Manager on each node separately.
+
+1. On the Harvester UI, go to **Hosts**.
+
+1. Locate the node in the list, and then select **⋮** > **Enable CPU Manager** or **Disable CPU Manager**.
+
+Allow some time for Harvester to apply the corresponding CPU Manager policy.
+
+![enable-cpu-manager](/img/v1.4/cpu-pinning/enable-cpu-manager.png)
+![disable-cpu-manager](/img/v1.4/cpu-pinning/disable-cpu-manager.png)
+
+### Limitations
+
+- The CPU Manager cannot be enabled on the witness node.
+
+- You can enable or disable the CPU Manager on only one management node at a time.
+
+- You must wait for an operation (enabling or disabling) to be completed before starting another.
+
+- VMs with CPU pinning enabled must be stopped before CPU Manager is disabled on the corresponding node.
+
+## Enable CPU Pinning on a New VM
+
+1. Verify that CPU Manager is enabled on one or more nodes.
+
+1. Go to **Virtual Machines**.
+
+1. Click **Create**.
+
+  ![create-vm](/img/v1.4/cpu-pinning/create-vm.png)
+
+1. On the **Advanced Options** tab, select **Enable CPU Pinning**.
+
+  ![vm-advanced-options](/img/v1.4/cpu-pinning/vm-advanced-options.png)
+
+1. On the **Advanced Options** tab, select **Enable CPU Pinning**.
+
+  ![vm-advanced-options](/img/v1.4/cpu-pinning/vm-advanced-options.png)
+
+1. Click **Save**.
+
+Enabling CPU pinning adds `dedicatedCpuPlacement: true` to `.spec.template.spec.domain.cpu` in the virtual machine configuration (YAML). When `dedicatedCpuPlacement` is set to `true`, the CPU and memory resource requests are automatically set to match the limits to ensure that the criteria for Guaranteed QoS are met.
+
+Because the requests and limits are identical, CPU and memory [Resource Overcommit](./resource-overcommit.md) settings do not apply to VMs with CPU pinning enabled.
+
+:::note
+
+To use CPU pinning on an existing VM, you must restart the VM after enabling the feature and saving the change.
+
+:::
+
+## VM Live Migration
+
+VMs with CPU pinning enabled can be migrated only if the CPU Manager is enabled on the target node. For more information, see [Live Migration](./live-migration.md).
+
+## Upgrades
+
+When upgrading a node, Harvester drains all pods and live migrates VMs to another node. To avoid interruptions to the upgrade process, ensure that the CPU Manager is enabled on other nodes and sufficient resources available whenever you use VMs with CPU pinning enabled.
```

Comment:
```suggestion
When upgrading a node, Harvester drains all pods and live migrates VMs to another node. To avoid interruptions to the upgrade process, ensure that the CPU Manager is enabled on other nodes and sufficient resources are available whenever you use VMs with CPU pinning enabled.
```